### PR TITLE
Test gh deploy

### PR DESCRIPTION
**Pull request by c2pig**
What does this PR do?
---------------------
To fix circleci config and to have status check for PR before merging

How can I test this?
--------------------
Not required.  Just login to circleci view master and rest of branch able to build by commit trigger

### CODE REVIEW

**Comment by c2pig** [comment]
@mengtzu , @ZhenYongSeekAsia.  I have wrong CI config in previous PR.  Error is fixed and tested now. 
 Need your help to review it.  Result available here : https://circleci.com/gh/seekinternational/seek-asia-style-guide/tree/test-gh-deploy

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,14 +32,10 @@ workflows:
...
       - build-deploy:
-        filters:
-          branches:
-            only:
-              - master
-
-        
+          filters:
+            branches:
+              ignore: master
```
Comment (mengtzu) [inline .circleci/config.yml]: Wouldn't only master be right?  We don't want to deploy random branches to gh pages

(`seek-style-guide` uses Surge to preview PRs, we might want to do that if we can figure a way to make it build out three.  But gh pages should only be master)

Comment (c2pig) [inline .circleci/config.yml]: @mengtzu Yes, i am trying to test whether certain steps in job can be reuse in other job or not.  So i have to add _ignore: master_ test it out.  Once it is done, i will rollback to _only: master_

Comment (c2pig) [inline .circleci/config.yml]: update to _only: master_

**Pull request merged**